src/changelog/2025/06/team-performance.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Memory Usage in Performance Feature
+description: "Enhance the Performance feature with memory usage metrics"
+date: 2025-06-5 11:43:32.0  
+authors: ["serban-costin"]
+tags:
+  - changelog
+issues:
+  - "https://github.com/FlowFuse/flowfuse/issues/5525"
+---
+
+We’ve expanded the **Performance** feature with an exciting new addition: memory usage metrics.
+
+### Key Features
+
+- **Memory Usage Insights**: Provides a **team-wide overview** of all hosted instances, giving users a comprehensive bird's eye view of the memory allocation and usage across a team.
+- **Enhanced Performance Understanding**: This addition ensures a holistic view of instance performance, aiding in better decision-making and upgrades.
+
+This improvement is part of our continuous effort to provide a better user experience and responds directly to community feedback.
+
+![Illustration of the New Feature](./images/team-performance.png){data-zoomable}
+_Overview of Memory Usage in the Performance Feature_
```

Comment:
This is the 3rd time now I've done a review and pointed out that this description doesn't describe the feature in the image or header of the changelog. The addition here is the team-level view, not the memory logging.